README.md

```diff
@@ -55,10 +55,23 @@ This repo follows [semantic versioning](https://semver.org/). Here are some exam
 
 ### Releasing
 
-**If you are not part of the Design System team, please submit a support request in #vfs-platform-support to the Design System Team requesting to create a new release with your changes.**
+Releases will occur no less often than at the beginning of each sprint (every other Wednesday), and may additionally be performed as-needed when critical bug fixes need to go out. Please reach out to us via #vfs-platform-support or #platform-design-system if you have a need for an unscheduled release.
 
 **For Design System Team only:**
-If the version numbers have been updated in `main` and you're ready to publish, [draft a new release](https://github.com/department-of-veterans-affairs/component-library/releases) where the tag follows the [`vX.Y.Z` semantic versioning structure](https://semver.org/). This tag should match the version found in `packages/core/package.json`. After creating a new tag, click the Generate release notes button. Verify the release notes look correct, and then click the Publish release button. Once a release is created, an attempt will be made to automatically publish the package(s) to npm using a GitHub action.
+
+1. If you are unsure if a new release should be created, check with the Release Manager and/or the rest of the team first, to make sure it's worth the effort at this time.
+2. Get the version number from `packages/core/package.json`, ensuring it's up-to-date and new.
```

Comment:
For my laziness, can we just make this `packages/core/package.json` a link to the actual file?  https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/core/package.json#L4